src/ophys_etl/transforms/classification.py

```diff
@@ -301,6 +277,119 @@ def downsample(trace: np.ndarray, input_fps: int, output_fps: int):
     return downsample
 
 
+def filter_excluded_rois(rois: List[SparseAndDenseROI]
+                         ) -> Tuple[List[SparseAndDenseROI],
+                                    List[SparseAndDenseROI]]:
+    """Filter ROIs by presence or absence of "exclusion_labels".
+
+    Parameters
+    ----------
+    rois : List[SparseAndDenseROI]
+        A list of ROIs.
+
+    Returns
+    -------
+    Tuple[List[SparseAndDenseROI], List[SparseAndDenseROI]]
+        A tuple of: (included_rois, excluded_rois)
+    """
+    included, excluded = [], []
+    for r in rois:
+        excluded.append(r) if r["exclusion_labels"] else included.append(r)
+    return included, excluded
+
+
+def _munge_traces(roi_data: List[SparseAndDenseROI], trace_file_path: str,
+                  trace_data_key: str, trace_names_key: str,
+                  trace_sampling_fps: int, desired_rate: int
+                  ) -> List[np.ndarray]:
+    """Read trace data from h5 file (in proper order) and downsample it.
+
+    Parameters
+    ----------
+    roi_data : List[SparseAndDenseROI]
+        A list of ROIs that conform to the SparseAndDenseROISchema.
+    trace_file_path : str
+        Path to h5 file containing trace data.
+    trace_data_key : str
+        Key used to access trace data.
+    trace_names_key : str
+        Key used to access the ROI name (id) corresponding to each trace.
+    trace_sampling_fps : int
+        Sampling rate (in Hz) of trace data.
+    desired_rate : int
```

Comment:
I'd prefer to see `desired_rate` and `trace_sampling_fps` have more parallel names, and, if units of Hz are specfied in 1 docstring, they should be specified in the other. In general `fps` doesn't make much sense for a trace. Hz should be sufficient. Or `rate`

Reply:
How about `trace_sampling_rate` and `desired_sampling_rate`. I will change the schemas fields: `trace_sampling_fps` and `downsample_to` to match as well?

Reply:
Sounds good. Yes, fps and Hz are the same, in camera speak. At this point, we're at a time series, pretty far away from a camera.

Reply:
Okay, I've made this change!